Mobile Buy SDK/Mobile Buy SDK/Operations/BUYCheckoutOperation.m

```diff
@@ -29,15 +29,15 @@
 #import "BUYClient+Internal.h"
 #import "BUYPaymentToken.h"
 #import "BUYRequestOperation.h"
+#import "BUYStatusOperation.h"
 
 @interface BUYCheckoutOperation ()
 
+@property (strong, nonatomic, readonly) BUYClient *client;
```

Comment:
This should probably not be a strong reference. Client is used only for communication. This object does not own it.